cookbook/security/custom_authentication_provider.rst

```diff
@@ -289,6 +289,18 @@ the ``PasswordDigest`` header value matches with the user's password.
     provider for the given token. In the case of multiple providers, the
     authentication manager will then move to the next provider in the list.
 
+.. note::
+
+    While the :phpfunction:`hash_equals` function was introduced in PHP 5.6,
+    you are safe to use it with any PHP version in your Symfony application. In
+    PHP versions prior to 5.6, `Symfony Polyfill`_ (which is included in
+    Symfony) will define the function for you.
+    
+    .. versionadded:: 2.8
```

Comment:
Versionadded is a very strange directive, as it can be used inside sentences. For this reason, empty lines after the directive start are not allowed (well they are, but the following text will be parsed as normal text instead of inside the vesrionadded directive).